backend/onedrive/onedrive.go

```diff
@@ -401,11 +423,11 @@ func Config(ctx context.Context, name string, m configmap.Mapper, config fs.Conf
 	region, graphURL := getRegionURL(m)
 
 	if config.State == "" {
+		accessScopes, _ := m.Get("access_scopes")
```

Comment:
This doesn't look right... `accessScopes` will be a string here, not a `[]string` so it needs parsing.

I suggest

```
var scopes fs.SpaceSepList
accessScopesString, _ := m.Get("access_scopes")
err :=  accessScopes.Set(accessScopesString)
if err != nil {
    return nil, fmt.Errorf("failed to parse access_scopes: %w", err)
}
oauthConfig.Scopes = []string(accessScopes)

Reply:
@ncw Thank you for the feedback. I've changed the var name from first line of your suggestion and added the code.